Remove version checks using pipdated.

Patch by Marius Brehler <marbre@linux.sungazer.de>
--- a/matplotlib2tikz/__init__.py
+++ b/matplotlib2tikz/__init__.py
@@ -15,7 +15,3 @@ from matplotlib2tikz.__about__ import (
         )
 
 from matplotlib2tikz.save import get_tikz_code, save
-
-import pipdated
-if pipdated.needs_checking(__name__):
-    print(pipdated.check(__name__, __version__))
